issue/types.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package issue;
+package flashlight.issue;
```

Comment:
I'm not sure if this change is needed as well @garmr-ulfr @myleshorton

Reply:
I think this just adds a scope to the definitions.  I'm not sure though.